crates/types/Cargo.toml

```diff
@@ -19,7 +19,7 @@ version = { workspace = true }
 [dependencies]
 anyhow = { workspace = true }
 derive_more = { version = "0.99" }
-fuel-vm-private = { workspace = true }
```

Comment:
Are we confident in enabling the debug feature for fuel-vm by default? It seems like there's a lot of extra code that gets enabled to check for breakpoints on every step etc.

Reply:
Yeah, it seems this part is heavy enough 
<img width="606" alt="image" src="https://github.com/FuelLabs/fuel-core/assets/18346821/3ee94799-0313-4030-9581-ec34c98cbc31">

We definitely need to optimize it on `fuel-vm` side.

I will create a separate PR in the `fuel-vm` repository. Meanwhile, I think we can merge this PR without waiting for the VM part.

Reply:
https://github.com/FuelLabs/fuel-vm/pull/554